rollout/bluegreen_test.go

```diff
@@ -420,50 +399,6 @@ func TestBlueGreenHandlePause(t *testing.T) {
 		assert.Equal(t, expectedPatch, rolloutPatch)
 	})
 
-	t.Run("SkipPreviewWhenActiveHasNoSelector", func(t *testing.T) {
```

Comment:
Test no longer needed as we always set the preview service to the NewRS podHash

Reply:
Actually removing this test regression coverage on this line: https://codecov.io/gh/argoproj/argo-rollouts/pull/217/src/rollout/bluegreen.go#L150. Renaming test to SkipPauseReconcileWhenActiveHasPodHashSelector